cypress/integration/smoke.spec.ts

```diff
@@ -42,13 +42,18 @@ e2e.scenario({
           name: dataSourceName,
           expectedAlertMessage: 'Data source is working',
           form: () => {
-            e2eSelectors.ConfigEditor.AuthenticationProvider.input().type('Access & secret key').type('{enter}');
```

Comment:
cool so are e2eSelectors on their way out or something?

Reply:
i was just changing this because the [e2eSelectors](https://github.com/grafana/athena-datasource/blob/2eb58a3f7459a9ac5cedf57c1a1e6616c7cd9d94/cypress/integration/smoke.spec.ts#L6) try to use `aria-label` to get the input field.

but after updating grafana-aws-sdk, the fields use the actual label to get the associated input element instead (from this [pr](https://github.com/grafana/grafana-aws-sdk-react/pull/85)). so I changed it to use `cy.contains` to get the label directly.